service/test/integration2_test.go

```diff
@@ -2961,3 +2961,11 @@ func TestPluginSuspendedBreakpoint(t *testing.T) {
 		cont("Continue 4", "plugin2.go", 9)
 	})
 }
+
+func TestRestart_createBreakpointWithID(t *testing.T) {
+	protest.AllowRecording(t)
+	withTestClient2("continuetestprog", t, func(c service.Client) {
+		_, err := c.CreateBreakpoint(&api.Breakpoint{ID: 2, FunctionName: "main.main", Line: 1, Name: "firstbreakpoint", Tracepoint: true})
```

Comment:
We should also validate the breakpoint gets the ID requested here, for completeness.

Reply:
Done.